src/components/views/rooms/EventTile.tsx

```diff
@@ -1611,8 +1618,8 @@ function isMessageEvent(ev: MatrixEvent): boolean {
 }
 
 export function haveTileForEvent(e: MatrixEvent, showHiddenEvents?: boolean): boolean {
-    // Only messages have a tile (black-rectangle) if redacted
-    if (e.isRedacted() && !isMessageEvent(e)) return false;
+    // Only show "Message deleted" tile for message or encrypted events
+    if (e.isRedacted() && !e.isEncrypted() && !isMessageEvent(e)) return false;
```

Comment:
I'm not sure I understand the need for this change, why would only encrypted events show that they were redacted?

Reply:
Previously, we showed the `Message removed` tile only for `m.room.message` events. But if an event is encrypted we can't tell if it's an `m.room.message` event, so we always show the tile for encrypted events

Reply:
Ah fun